test/server.test.js

```diff
@@ -0,0 +1,10 @@
+const request = require('supertest');
+const app = require('../app');
+
+describe('Health Check', () => {
+  it('should return a JSON object with property "uptime"', async () => {
+    const res = await request(app).get('/health');
+    expect(res.statusCode).toEqual(200);
+    expect(res.body).toHaveProperty('uptime');
+  });
+});
```

Comment:
Let's add one more test that `/` returns a 200 and a content-type of text/html...